portfolio/src/main/webapp/gallery.html

```diff
@@ -21,19 +22,19 @@
 <nav class="navbar navbar-inverse">
   <div class="container-fluid">
     <div class="navbar-header">
-      <a class="navbar-brand" href="#">Madeline Cuozzo</a>
+      <a class="navbar-brand" href="#" style="font-family: Lekton">Madeline Cuozzo</a>
```

Comment:
Update the style sheet to add font to an existing class. Can't quite tell if you want the same font all over or not, you could put it in higher level container class, if that is so.